package-overrides/github/brianreavis/selectize.js@0.12.0.json

```diff
@@ -0,0 +1,19 @@
+{
+  "directories": {
+    "lib": "dist"
+  },
+  "main": "js/selectize",
+  "registry": "jspm",
+  "dependencies": {
+    "jquery": "*",
+    "sifter": "*",
```

Comment:
Excuse me, but what you mean by "other versions"? You mean jquery, sifter, microplugin?

Selectize docs say it requires jQuery >=1.7, no version requirements for sifter and microplugin. Does jspm supports `"jquery": ">=1.7"`? I thought not...